topics/statistics/tutorials/intro-to-ml-with-python/tutorial.md

```diff
@@ -11,7 +11,7 @@ requirements:
   - python-basics
   - python-warmup-stat-ml
 questions:
-- to do
+- how can we use Machine-Learning to make more generalizable models?
 objectives:
 - general sklearn syntax intro
 - overfit/underfit
```

Comment:
would be great to write the objectives as phrases, preferably using [Bloom's taxonomy](https://tips.uark.edu/using-blooms-taxonomy/#gsc.tab=0), for example "Understanding the concepts of overfitting/underfitting and how to avoid them" (or whatever makes sense for your tutorial).

Basically, complete the sentence "At the end of this tutorials you will be able to .."  (Understand the concept of overfitting/underfitting... for example)